cf/client.go

```diff
@@ -31,7 +31,7 @@ func NewClient(config *ClientConfiguration) (*PlatformClient, error) {
 	}
 	return &PlatformClient{
 		Client: cfClient,
-		reg:      config.Reg,
+		reg:    config.Reg,
```

Comment:
reg settings are now used on two places - once as app. in the proxy framework and once as reg. in the cf proxy - can we avoid this somehow

Reply:
these are scoped in cf so you would have to set both app: and cf.app: with exactly the same values which bring some confusion i think.